modules/storages/lib/open_project/storages/engine.rb

```diff
@@ -44,6 +44,7 @@ class Engine < ::Rails::Engine
       OpenProject::FeatureDecisions.add :storage_file_picking_select_all
       OpenProject::FeatureDecisions.add :storage_project_folders
       OpenProject::FeatureDecisions.add :managed_project_folders
+      OpenProject::FeatureDecisions.add :automatically_managed_project_folders
```

Comment:
this is duplicate, isn't it? `:managed_project_folders` was already defined for the scope of anything that goes for the automatically managed folders.

Reply:
IIRC we discussed and agreed to define a dedicated flag as this is in the global administration. cc @wielinde